test/zzz-duplicate-entries.sh

```diff
@@ -68,9 +68,8 @@ wait_for_fluentd_to_catch_up
 
 es_pod=$( get_es_pod es )
 for message in $MESSAGES; do
-  full_msg="GET /${message} 404 "
-  qs='{"query":{"match_phrase":{"message":"'"${full_msg}"'"}}}'
-  if os::cmd::try_until_text "curl_es ${es_pod} /project.logging.*/_count -X POST -d '$qs' | get_count_from_json_from_search" 1 $(( 10 * minute )); then
+  qs='{"query":{"term":{"systemd.u.SYSLOG_IDENTIFIER":"'"${message}"'"}}}'
+  if os::cmd::try_until_text "curl_es ${es_ops_pod} /.operations.*/_count -X POST -d '$qs' | get_count_from_json_from_search" 1 $(( 10 * minute )); then
```

Comment:
Use `get_count_from_json` because you are doing a `_count` operation as opposed to a `_search` operation for which you would use `get_count_from_json_from_search`